cvat/apps/engine/serializers.py

```diff
@@ -726,6 +748,13 @@ def validate_job_file_mapping(self, value):
 
         return value
 
+    def validate_tus_file_order(self, value):
```

Comment:
This function is not used anywhere.

And I don't think it should be - as long as you verify that the list of filenames matches the actually uploaded filenames, it will follow that there are no suspicious paths in it.